codebasin/walkers/tree_mapper.py

```diff
@@ -1,12 +1,12 @@
-# Copyright (C) 2019 Intel Corporation
+# Copyright (C) 2019-2024 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 
-import logging
 import collections
+import logging
 
-from .tree_walker import TreeWalker
+from codebasin.walkers.tree_walker import TreeWalker
 
-log = logging.getLogger('codebasin')
+log = logging.getLogger("codebasin")
```

Comment:
Is this even used?

Reply:
As discussed offline, this might (although it's _very_ unlikely) be something that somebody is relying on.

I double-checked, and as far as I can tell, none of these logging statements were introduced as new code -- in some cases they got reordered, and in all cases they moved from `'` to `"`, so the fact they were there before might not be obvious from the diff.

I think we should leave them for now, and remove them in 2.0.0.